README.md

```diff
@@ -57,6 +57,20 @@ cyclonedx-bom -o bom.xml
 cyclonedx-bom -o bom.json
 ```
 
+## Usage with docker
+
+Run `cyclonedx/cyclonedx-node` docker image inside your project folder using:
+
+```bash
+docker run --rm \
+  -v `pwd`:/src \
+  -w /src \
+  cyclonedx/cyclonedx-node -o /src/bom.xml /src
```

Comment:
i'd recomment to remove `/src` from the arguments here, since `/src` is already set as the working-dir (`-w /src`)

```diff
- cyclonedx/cyclonedx-node -o /src/bom.xml /src
+ cyclonedx/cyclonedx-node -o bom.xml
```

Reply:
Yes, good point.